src/parseSet.java

```diff
@@ -7,50 +7,51 @@
 class parseSet {
 
     private static final String DELIMITER = "[^a-zA-Z0-9_,]+";
-    private static final String FIRSTPATH = "Text/FIRST.txt";
```

Comment:
Curious if there's a reason we're changing from a table?

This new version seems easier to read IMO so I'm for it, but just wondering if there's another reason?

also, can we go ahead and remove Text/FIRST.txt and Text/PREDICT.txt ?

Reply:
The diffs on merges were not helpful. I assume the new version would be easier to maintain (haha) and more compatible with diff.

We can go ahead and remove the old, unused files. I kept them in case the data was not transferred correctly.